### ISSUE TITLE
Error on process kill when babel-watch is run using npm

### ISSUE THREAD

**Issue by coreyleelarson** [kmagiera/babel-watch#63]
**node v6.11.1**
**npm v5.3.0**
**babel-watch v2.0.7**

This doesn't appear to cause any problems, but it's quite annoying.

When I run babel-watch with ```npm run serve:watch``` and kill the process with <kbd>CTRL</kbd>+<kbd>C</kbd>, I get the following error:

**package.json**
```javascript
{
  scripts: {
    "serve:watch": "cross-env NODE_ENV=development babel-watch server/development.js"
  }
}
```

**terminal command run**
```bash
npm run serve:watch
```

**terminate command with**
<kbd>CTRL</kbd>+<kbd>C</kbd>

**terminal output**
```bash
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-ssr-boilerplate@1.0.0 serve:watch: `cross-env NODE_ENV=development babel-watch server/development.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-ssr-boilerplate@1.0.0 serve:watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/nmcl54/.npm/_logs/2017-08-07T06_09_19_750Z-debug.log
```

Interestingly, I do not get this error when I run my script with ```yarn run serve:watch```.

**Comment by coreyleelarson** [comment]
The process is exiting with a failure on the SIGINT event at line 120:

**babel-watch.js**
```javascript
process.on('SIGINT', function() {
  watcher.close();
  process.exit(1);
});
```

Is there a specific reason for this or can it be changed to ```process.exit(0);```

**Comment by coreyleelarson** [comment]
https://github.com/kmagiera/babel-watch/pull/64

**Comment by STRML** [comment]
Looks like this is fixed by #87.
